### ISSUE TITLE
"vue/attribute-hyphenation" ignore is not working

### ISSUE THREAD

**Issue by sarmadbokhari** [vuejs/eslint-plugin-vue#496]
**Tell us about your environment**

**ESLint Version:** 4.19.1
**eslint-plugin-vue Version:** 4.5.0
**Node Version:** 9.11.1

**Please show your full configuration:**
```
// https://eslint.org/docs/user-guide/configuring

module.exports = {
  env: {
    "node": true,
    "es6": true,
    "mocha": true
  },
  parserOptions: {
    "ecmaVersion": 8,
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true
    }
  },
  // https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
  // consider switching to `plugin:vue/strongly-recommended` or `plugin:vue/recommended` for stricter rules.
  extends: ["plugin:vue/strongly-recommended"],
  // required to lint *.vue files
  plugins: [
    "vue"
  ],
  rules: {
    "indent": [
      "error",
      2
    ],
    "linebreak-style": [
      "error",
      "unix"
    ],
    "quotes": [
      "error",
      "single"
    ],
    "keyword-spacing": [
      "error"
    ],
    "object-curly-spacing": [
      "error",
      "always"
    ],
    "no-trailing-spaces": [
      "error"
    ],

    // eslint-plugin-vue overrides
    "vue/attribute-hyphenation": [2, "never", {
      "ignore": ["enter-active-class"]
    }],
    "vue/attributes-order": ["error", {
      "order": ["DEFINITION", "LIST_RENDERING", "CONDITIONALS", "RENDER_MODIFIERS", "GLOBAL", "UNIQUE", "BINDING", "OTHER_ATTR", "EVENTS", "CONTENT"]
    }],
    "vue/max-attributes-per-line": ["error", {
      "singleline": 2,
      "multiline": {
        "max": 2,
        "allowFirstLine": true
      }
    }],
    "vue/require-default-prop": "off",
  }
};
```
**What did you expect to happen?**
I'm expecting this to work as defined in the docs
```
    "vue/attribute-hyphenation": [2, "never", {
      "ignore": ["enter-active-class"]
    }],
```
**What actually happened? Please include the actual, raw output from ESLint.**
```
	Configuration for rule "vue/attribute-hyphenation" is invalid:
	Value ["never",{"ignore":["enter-active-class"]}] should NOT have more than 1 items.
```

**Comment by tronjs** [comment]
same here. Specifying `{"ignore":["some-things"]}` but it all says `should NOT have more than 1 items`

**Comment by mochetts** [comment]
Having the same issue

**Comment by mochetts** [comment]
Seems that it was fixed as part of this PR #461

**Comment by michalsnik** [comment]
Please try `v4.7.0` :)
